jobs/datadog-firehose-nozzle/templates/datadog-firehose-nozzle.json.erb

```diff
@@ -56,7 +66,7 @@ JSON.pretty_generate(
     "FlushMaxBytes" => p("datadog.flush_max_bytes"),
     "HTTPProxyURL": p("nozzle.http_proxy_url", ""),
     "HTTPSProxyURL": p("nozzle.https_proxy_url", ""),
```

Comment:
Hmm, so should we also change these two to use `=>` instead of `:`?

Reply:
done